src/components/NurseryWithdrawals/WithdrawalDetails/WithdrawalTabPanelContent.tsx

```diff
@@ -41,19 +53,23 @@ export default function WithdrawalTabPanelContent({
     },
     {
       title: strings.QUANTITY,
-      data: withdrawalSummary?.totalWithdrawn?.toString() ?? '',
+      data: delivery?.plantings?.reduce((acc, planting) => acc + planting.numPlants, 0)?.toString() ?? '',
     },
     {
       title: strings.FROM_NURSERY,
       data: facilityName ?? '',
     },
     {
       title: strings.DESTINATION,
-      data: withdrawalSummary?.destinationName ?? '',
+      data: siteName,
     },
     {
       title: strings.TO_PLOT,
-      data: withdrawalSummary?.plotNames ?? '',
+      data:
```

Comment:
We were showing the plot name before and now just the id, is that intentional ?

Reply:
Yeah, the search result was nice in that it had a lot of this information populated in the summary. I will take a look at the `getPlantingSiteData()` method though as well.